src/table/body/table-row.component.ts

```diff
@@ -19,6 +19,7 @@ import { Observable } from "rxjs";
 			<td
 				*ngIf="model.hasExpandableRows()"
 				ibmTableExpandButton
+				class="bx--table-expand-v2"
```

Comment:
do we really want to use a version number in the classname?

Reply:
It's the class name defined in the carbon components css. I needed to use the v2 class because there was styling that the css ignores if it is v2 but doesn't ignore if its the regular bx--table-expand.

Reply:
🤦‍♂ I guess we can't do much there then